docs/content/en/schemas/v1beta5.json

```diff
@@ -326,7 +326,7 @@
         },
         "image": {
           "type": "string",
-          "description": "Docker image used by the Kaniko pod. Defaults to the latest released version of <code>gcr.io/kaniko-project/executor</code>."
+          "description": "Docker image used by the Kaniko pod. Defaults to the v1beta5 released version of <code>gcr.io/kaniko-project/executor</code>."
```

Comment:
@balopat this is not supposed to be replaced :-)

Reply:
LOL nice catch!!!!